README.md

```diff
@@ -43,6 +43,7 @@ serve ~/myproj
 - `sudo` may be necessary.
 - If a static path is not provided the current directory content will be served.
 - You can change the **port** setting the `PORT` environmental variable: `PORT=4433 serve ~/myproj`. Specifying port number will also prevent http to https redirect.
+- You can change the **domain** setting the `DOMAIN` environmental variable: `DOMAIN=192.168.0.2 serve ~/myproj`.
```

Comment:
The variable is called DOMAIN but you are passing an IP address. What about calling it host or change the example?

Reply:
Yes your right. Host is probably a better naming.